infra/main.bicepparam

```diff
@@ -1,8 +1,8 @@
 using 'main.bicep'
 
-param environmentName = readEnvironmentVariable('AZURE_ENV_NAME', '')
-param location = readEnvironmentVariable('AZURE_LOCATION', 'eastus')
+param environmentName = readEnvironmentVariable('AZURE_ENV_NAME', 'env_name')
+param location = readEnvironmentVariable('AZURE_LOCATION', 'location')
 param openAiHost = readEnvironmentVariable('OPENAI_HOST', 'azure')
-param openAiApiKey = readEnvironmentVariable('OPENAI_API_KEY', '')
+param openAiApiKey = readEnvironmentVariable('OPENAI_API_KEY', 'openai_api_key')
```

Comment:
Do we need any defaults here?  THose defaults won't work so, why not remove them?

Reply:
Removed.